Test failure in ydb/tests/compatibility/test_rolling.py.TestRolling.test_kv[rolling_stable-25-4-1_to_prestable-26-1-column]

ydb/tests/compatibility/test_rolling.py.TestRolling.test_kv[rolling_stable-25-4-1_to_prestable-26-1-column]

**Test Failure**

**Build Information:**
- Version: 5b365c8c66bb6df0b1f822d21a5fa92f5943e9b2
- Build Preset: relwithdebinfo
- Branch: main

**Test Report:** [View report](https://storage.yandexcloud.net/ydb-gh-logs/ydb-platform/ydb/Regression-run_compatibility/21834015376/ya-main-x86-64/try_1/ya-test.html#FAIL)

**Run Command:**
```
./ya make -ttt --build "relwithdebinfo" -F 'test_rolling.py::TestRolling::test_kv[rolling_stable-25-4-1_to_prestable-26-1-column]' ydb/tests/compatibility
```

**Owner:** [TEAM:@ydb-platform/engineering](https://github.com/orgs/ydb-platform/teams/engineering)

**Success rate (last 4 days):**
MAIN: 64.3% (9/14)

**Test run history:** [link](https://datalens.yandex/34xnbsom67hcq?branch=main&full_name=ydb/tests/compatibility/test_rolling.py.TestRolling.test_kv[rolling_stable-25-4-1_to_prestable-26-1-column]&build_type=relwithdebinfo)

More info in [dashboard](https://datalens.yandex/4un3zdm0zcnyr)

**Logs:**
- [log](https://storage.yandexcloud.net/ydb-gh-logs/ydb-platform/ydb/Regression-run_compatibility/21834015376/ya-main-x86-64/try_1/artifacts/logs/ydb/tests/compatibility/test-results/py3test/testing_out_stuff/test_rolling.py.TestRolling.test_kv.rolling_stable-25-4-1_to_prestable-26-1-column.log)
- [logsdir](https://storage.yandexcloud.net/ydb-gh-logs/ydb-platform/ydb/Regression-run_compatibility/21834015376/ya-main-x86-64/try_1/test_artifacts/ydb-tests-compatibility.zip)

**Error Summary:**
```
ydb/tests/compatibility/test_rolling.py:98: in test_kv
    for _ in self.roll():
ydb/tests/library/compatibility/fixtures.py:310: in roll
    self._wait_for_readiness()
ydb/tests/library/compatibility/fixtures.py:238: in _wait_for_readiness
    self.driver = self.create_driver()
ydb/tests/library/compatibility/fixtures.py:233: in create_driver
    driver.wait(timeout=60)
contrib/python/ydb/py3/ydb/pool.py:418: in wait
    self._store.subscribe().result(timeout)
contrib/tools/python3/Lib/concurrent/futures/_base.py:458: in result
    raise TimeoutError()
E   TimeoutErrorteardown failed:
ydb/tests/compatibility/test_rolling.py:18: in setup
    yield from self.setup_cluster(
ydb/tests/library/compatibility/fixtures.py:297: in setup_cluster
    self.cluster.stop()
ydb/tests/library/harness/kikimr_runner.py:740: in stop
    raise daemon.SeveralDaemonErrors(saved_exceptions)
E   ydb.tests.library.harness.daemon.SeveralDaemonErrors: Daemon failed with message: Unexpectedly finished before stop.
E   Process exit_code = -6.
E   Stdout file name: 
E   /home/runner/actions_runner/_work/ydb/ydb/tmp/out/ydb/tests/compatibility/test-results/py3test/testing_out_stuff/test_rolling.py.TestRolling.test_kv.rolling_stable-25-4-1_to_prestable-26-1-column/cluster/node_1/stdout
E   Stderr file name: 
E   /home/runner/actions_runner/_work/ydb/ydb/tmp/out/ydb/tests/compatibility/test-results/py3test/testing_out_stuff/test_rolling.py.TestRolling.test_kv.rolling_stable-25-4-1_to_prestable-26-1-column/cluster/node_1/stderr
E   Stderr content:
E   
E   Unknown grpc service "nbs" was not enabled!
E   Unknown grpc service "nbs" was not enabled!
E   GRpc memory quota was set but disabled due to issues with grpc quoter, to enable it use EnableGRpcMemoryQuota option
E   Current KQP shutdown state: spent 0 seconds, 2 sessions to shutdown
E   Current KQP shutdown state: spent 3.002122 seconds, 1 sessions to shutdown
E   Current KQP shutdown state: spent 6.004255 seconds, 1 sessions to shutdown
E   Current KQP shutdown state: spent 9.006281 seconds, 1 sessions to shutdown
E   Drain completed.
E   GRpc memory quota was set but disabled due to issues with grpc quoter, to enable it use EnableGRpcMemoryQuota option
E   VERIFY failed (2026-02-09T20:03:31.352785Z): tablet_id=72075186224037889;tx_state=TTxProgressTx::Execute;tx_current=0;tx_id=281474976754062;commit_tx_id=281474976754062;commit_lock_id=281474976754061;verification=!lock.IsBroken();fline=manager.cpp:119;error=the tx has writes, it is broken, and we are committing it;writes_count=1;
E   ydb/library/actors/core/log.cpp:857
E   ~TVerifyFormattedRecordWriter(): requirement false failed
E   0. /-S/util/system/yassert.cpp:86: InternalPanicImpl @ 0xA11E70A
E   1. /-S/util/system/yassert.cpp:55: Panic @ 0xA118D0B
E   2. /tmp//-S/ydb/library/actors/core/log.cpp:857: ~TVerifyFormattedRecordWriter @ 0xAD0E6AF
E   3. /tmp//-S/ydb/core/tx/columnshard/operations/manager.cpp:119: CommitTransactionOnExecute @ 0x13B7ADFB
E   4. /-S/ydb/core/tx/columnshard/transactions/operators/ev_write/abstract.h:96: ProgressOnExecute @ 0x9B64D96
E   5. /tmp//-S/ydb/core/tx/columnshard/columnshard__progress_tx.cpp:79: Execute @ 0x13C6154D
E   6. /tmp//-S/ydb/core/tablet_flat/flat_executor.cpp:2036: ExecuteTransaction @ 0xC588155
E   7. /tmp//-S/ydb/library/actors/core/actor.cpp:350: Receive @ 0xACBAF24
E   8. /tmp//-S/ydb/library/actors/core/executor_thread.cpp:267: Execute @ 0xAD00405
E   9. /tmp//-S/ydb/library/actors/core/executor_thread.cpp:455: operator() @ 0xAD04004
E   10. /tmp//-S/ydb/library/actors/core/executor_thread.cpp:507: ProcessExecutorPool @ 0xAD03C10
E   11. /tmp//-S/ydb/library/actors/core/executor_thread.cpp:533: ThreadProc @ 0xAD047A6
E   12. /-S/util/system/thread.cpp:245: ThreadProxy @ 0xA120027
E   13. ??:0: ?? @ 0x7F97E04DBAC2
E   14. ??:0: ?? @ 0x7F97E056D8BF
```

--- a/ydb/core/tx/columnshard/operations/manager.cpp
+++ b/ydb/core/tx/columnshard/operations/manager.cpp
@@ -115,9 +115,6 @@ void TOperationsManager::CommitTransactionOnExecute(
     auto& lock = GetLockFeaturesForTxVerified(txId);
     TLogContextGuard gLogging(
         NActors::TLogContextBuilder::Build(NKikimrServices::TX_COLUMNSHARD_TX)("commit_tx_id", txId)("commit_lock_id", lock.GetLockId()));
-    if (lock.GetWriteOperations().size() > 0) {
-        AFL_VERIFY(!lock.IsBroken())("error", "the tx has writes, it is broken, and we are committing it")("writes_count", lock.GetWriteOperations().size());
-    }
     TVector<TWriteOperation::TPtr> commited;
     for (auto&& opPtr : lock.GetWriteOperations()) {
         opPtr->CommitOnExecute(owner, txc, snapshot);

